hermes_bridge/callbacks.py

```
@@ -131,9 +301,61 @@ def clarify_callback(question=None, **kwargs):
+        line = json.dumps(payload, ensure_ascii=False)
+        with self._lock:
+            self._fh.write(line)
+            self._fh.write("\n")
+            self._fh.flush()
+
+    def close(self) -> None:
+        with self._lock:
+            try:
+                self._fh.close()
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)